### ISSUE TITLE
how can I implement Grad_CAM on object detection model ??

### ISSUE THREAD

**Issue by sihan-shao** [kazuto1011/grad-cam-pytorch#21]
I find all code about Grad-CAM on github to be aimed classification model, so how can I implement Grad_CAM on object detection model that has two tasks: classification and regression??

**Comment by kazuto1011** [comment]
I think the problem is how to compute weights to merge the activation maps. The original Grad-CAM employs gradients of a class-specific score, which can be adapted to the bbox-wise classification. For the regression part, you can see [the Keras example](https://github.com/jacobgil/keras-steering-angle-visualizations) which computes the gradients of the predicted value.
